lib/states_and_countries.rb

```diff
@@ -0,0 +1,308 @@
+module StatesAndCountries
```

Comment:
Why make this a module and not a class?

Reply:
You don't really need state or initialization with this so probably makes more sense to keep it lightweight and composable.